nix/default.nix

```diff
@@ -43,6 +43,11 @@ let
           }
         )
 
+        ( self: super: {
```

Comment:
this is not needed any more: https://search.nixos.org/packages?channel=21.11&from=0&size=50&sort=relevance&type=packages&query=nix-update